src/Controller/PlayerController.php

```diff
@@ -733,6 +733,33 @@ public function getRecentBadges() {
     return $response;
   }
 
+  public function all_player_summary($uid = 0){
+    $user = \Drupal::currentUser();
+    if (!$user->isAuthenticated()) {
+      return new RedirectResponse('/user/login?destination=/summergame/player');
+    }
+    $roles = $user->getRoles();
+     if ((int)$user->id() !== (int)$uid && !in_array('staff', $roles) && !in_array('administrator', $roles)) {
+        return new RedirectResponse('/user/login?destination=/summergame/player');
+    }
+    $players = summergame_player_load_all($uid);
+    foreach ($players as $key => $player) {
+      $player_points = summergame_get_player_points($player['pid']);
+      $commerce_points = commerce_summergame_get_player_balances($player['pid']);
+      $players[$key]['player_points'] = $player_points;
+      $players[$key]['commerce_points'] = $commerce_points;
+    }
+    $renderArray = array(
+      "players"=>$players,
+    );
+    $module_path = \Drupal::service('module_handler')->getModule('summergame')->getpath();
```

Comment:
Can add a new entry to the summergame_theme() hook and pass in a single "players" variable to get it to automatically load the twig template rather than pulling it out of the filesystem and rendering the full html here in the controller